src/node/txreconciliation.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <node/txreconciliation.h>
+
+#include <util/system.h>
+
+#include <unordered_map>
+#include <variant>
+
+
+namespace {
+
+/** Static salt component used to compute short txids for sketch construction, see BIP-330. */
+static const std::string RECON_STATIC_SALT = "Tx Relay Salting";
+static const HashWriter RECON_SALT_HASHER = TaggedHash(RECON_STATIC_SALT);
```

Comment:
the static keyword is redundant in unnamed namespaces
```suggestion
const std::string RECON_STATIC_SALT = "Tx Relay Salting";
const HashWriter RECON_SALT_HASHER = TaggedHash(RECON_STATIC_SALT);
```